snippets/kotlin.json

```diff
@@ -59,5 +66,50 @@
             "\t${0}",
             "}"
         ]
+    },
+    "tryCatch": {
```

Comment:
Instead of creating a new block, would it better to add a `catch` block to the existing `try` snippet? I don't think anyone really writes a `try` without an accompanying `catch` block